config/plugin/network_problem.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
 # This plugin checks for common network issues. Currently, it only checks
-# if the conntrack table is full. 
+# if the conntrack table is full.
 
 OK=0
 NONOK=1
 UNKNOWN=2
 
-[ -f /proc/sys/net/ipv4/netfilter/ip_conntrack_max ] || exit $UNKNOWN
-[ -f /proc/sys/net/ipv4/netfilter/ip_conntrack_count ] || exit $UNKNOWN
+# "nf_conntrack" replaces "ip_conntrack" - support both
+NF_CONNTRACK_COUNT='/proc/sys/net/netfilter/nf_conntrack_count'
+NF_CONNTRACK_MAX='/proc/sys/net/netfilter/nf_conntrack_max'
+IP_CONNTRACK_COUNT='/proc/sys/net/ipv4/netfilter/ip_conntrack_count'
+IP_CONNTRACK_MAX='/proc/sys/net/ipv4/netfilter/ip_conntrack_max'
 
-conntrack_max=$(cat /proc/sys/net/ipv4/netfilter/ip_conntrack_max)
-conntrack_count=$(cat /proc/sys/net/ipv4/netfilter/ip_conntrack_count)
+if [ -f $NF_CONNTRACK_COUNT ] && [ -f $NF_CONNTRACK_MAX ]; then
```

Comment:
nit: Could you help change the `[ -f xxx ]`s here and below into `[[ -f xxx ]]`? Thanks! :)
This is because `[[ ]]` is [more prefered](https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B) than `[ ]`. If you could help use this chance to help improve the code quality, I'm sure everyone would appreciate it :)

Reply:
Updated to use `[[ ]]`, I also switched to 2 space indent to follow Google Shell Style Guide.

I think same should be done for `check_ntp.sh`.

Reply:
Thanks! Yes I agree, we should update `check_ntp.sh` as well as a small improvement.